packages/fast-usdc/src/exos/advancer.js

```diff
@@ -212,15 +212,22 @@ export const prepareAdvancerKit = (
          * @param {Error} error
          * @param {AdvancerVowCtx & { tmpSeat: ZCFSeat }} ctx
          */
-        onRejected(error, { tmpSeat }) {
-          // TODO return seat allocation from ctx to LP?
-          log('🚨 advance deposit failed', q(error).toString());
-          // TODO #10510 (comprehensive error testing) determine
-          // course of action here
+        onRejected(error, { tmpSeat, advanceAmount, ...restCtx }) {
+          // we don't expect this to be a common failure. if it happens, return funds to LP
           log(
-            'TODO live payment on seat to return to LP',
-            q(tmpSeat).toString(),
+            '⚠️ deposit to localOrchAccount failed, attempting to return payment to LP',
+            q(error).toString(),
```

Comment:
why stringify the error? why not let `log` do that?

this will lose stack info, no?

Reply:
Good point, done